index.html

```diff
@@ -329,21 +329,30 @@ <h2><dfn>ParityType</dfn> enum</h2>
 This section defines the <dfn>Security considerations</dfn> for the <cite>
 Serial API</cite>.
 
-It is RECOMMENDED that an user interface be presented to the user to allow them
-to select the serial ports that the page can access. Such an interface would
-allow an end user to select zero ore more serial ports on the device - or to
-reject the request to access any ports.
+It is RECOMMENDED that a user interface be presented to the user to allow them
+to select the serial ports that the page can access. Such an interface SHOULD
+allow an end user to select a serial port on the device or to reject the request
+to access any ports.
+
+The interface SHOULD represent serial ports in a way meaningful to the user.
+(For example, device serial numbers would not likely be meaningful, at least not
+on their own.) Standard names, like `COM1`, and device manufacturer names, are
+more likely to be meaningful.
```

Comment:
I wonder if there is any industry best practice around naming these things? We could suggest implementers use those. There might not be, but just wondering...